.env.example

```
@@ -1,6 +1,45 @@
 # Example environment variables for Omega_KG
 # Copy this file to .env and fill in the actual values
 
+# APP_ENV can be 'development' or 'production'
 APP_ENV=development
-DATABASE_URL=postgresql://user:password@localhost:5432/omega_kg
-SECRET_KEY=your-secret-key
+# Neo4j connection URI (format: bolt://host:port)
+NEO4J_URI=bolt://localhost:7687
+NEO4J_USER=neo4j
+NEO4J_USER=neo4j
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEO4J_USER key is duplicated

(DuplicatedKey)